README.markdown

```diff
@@ -1,9 +1,13 @@
 # fortune.py
 
+This is a fork of
+
 [http://github.com/goerz/fortune.py](http://github.com/goerz/fortune.py)
 
 Author: [Michael Goerz](http://michaelgoerz.net)
 
+with support for Python3 (3.7.1) and UTF8 fortune cookie files (german, chinese etc.)
```

Comment:
Since this is just a simple script, and people may want to run it on some old cluster where only Python 2 is available, maybe we can make sure that it runs with *both* Python 2 and Python 3? (relying on `six` is fine)

Reply:
Well, I don't think I can do that since I don't have mich know how on Python 2 and 3.

Reply:
Fair enough, but I think then it's better to just keep your version as a fork.